renderdoc/driver/ihv/intel/intel_gl_counters.cpp

```diff
@@ -104,6 +104,25 @@ void IntelGlCounters::addCounter(const IntelGlQuery &query, GLuint counterId)
   counter.desc.resultType = glToRdcCounterType(counter.dataType);
   counter.desc.unit = CounterUnit::Absolute;
 
+  if(counter.desc.description.contains("Unit: cycles."))
+    counter.desc.unit = CounterUnit::Cycles;
+  else if(counter.desc.description.contains("Unit: bytes."))
+    counter.desc.unit = CounterUnit::Bytes;
+  else if(counter.desc.description.contains("Unit: percent."))
+    counter.desc.unit = CounterUnit::Percentage;
+  else if(counter.desc.description.contains("Unit: Hz."))
+    counter.desc.unit = CounterUnit::Hertz;
+  else if(counter.desc.description.contains("Unit: ns."))
+  {
```

Comment:
Style nit: if you need braces for any branch of an if/else chain please add it for all of them.